codes/rust/chapter_hashing/hash_map_chaining.rs

```diff
@@ -102,17 +102,15 @@ impl HashMapChaining {
         // 遍历桶，若遇到指定 key ，则更新对应 val 并返回
         for pair in bucket {
             if pair.key == key {
-                pair.val = val.clone();
+                // replace value
```

Comment:
Plz do not add the unnecessary comments for the consistency between multiple languages

Reply:
ok

Reply:
updated